package.json

```diff
@@ -20,45 +20,45 @@
   "scripts": {
     "test:unit": "mocha -r ts-node/register --reporter spec tests/**/*.spec.ts",
     "test:e2e": "ts-node tests/e2e/test.ts",
-    "lint": "resin-lint --typescript lib/ && tsc --noEmit",
+    "lint": "balena-lint --typescript lib tests && tsc --noEmit",
     "test": "npm run build && npm run lint && npm run test:unit && npm run test:e2e",
     "build": "rimraf ./build && npm run prettify && tsc && npm run readme",
     "prepublish": "require-npm4-to-publish",
     "prepublishOnly": "npm test",
     "prepare": "npm run build",
-    "prettify": "prettier --config ./node_modules/resin-lint/config/.prettierrc --write \"lib/**/*.ts\" \"tests/**/*.ts\"",
+    "prettify": "balena-lint --typescript --fix lib tests",
     "readme": "jsdoc2md --template doc/README.hbs build/settings.js > README.md"
   },
   "author": "Juan Cruz Viotti <juan@balena.io>",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@resin.io/types-mochainon": "2.0.1",
+    "@balena/lint": "^5.0.4",
     "@resin.io/types-wary": "1.1.1",
-    "@types/bluebird": "3.5.20",
-    "@types/common-tags": "1.4.0",
-    "@types/mocha": "2.2.48",
-    "@types/node": "^8.10.10",
-    "bluebird": "^3.5.0",
-    "common-tags": "^1.4.0",
-    "husky": "^3.0.5",
+    "@types/bluebird": "^3.5.30",
+    "@types/chai": "^4.2.11",
+    "@types/chai-as-promised": "^7.1.2",
+    "@types/common-tags": "^1.8.0",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^8.10.60",
+    "bluebird": "^3.7.2",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "common-tags": "^1.8.0",
+    "husky": "^4.2.5",
     "jsdoc-to-markdown": "^5.0.1",
-    "lint-staged": "^9.2.5",
-    "mocha": "^6.2.0",
-    "mochainon": "^2.0.0",
```

Comment:
I removed mochainon as it doesn't play that nicely with the chai-no-unused-expressions and using the chai modules directly allows us to easily update them and use official typings